src/ui.form-checkbox/config.ts

```diff
@@ -1,11 +1,14 @@
 export default /*tw*/ {
-  checkboxLabel: "{ULabel}",
+  checkboxLabel: {
+    base: "{ULabel}",
+    label: "hover:cursor-pointer",
```

Comment:
bad solution. need to redefine it at all components with `<ULabel>` need to be fixed inside `<ULabel>`.